src/system/SystemPacketBuffer.h

```diff
@@ -142,6 +144,20 @@ class DLL_EXPORT PacketBuffer : private pbuf
 #endif // CHIP_SYSTEM_CONFIG_USE_LWIP
 
 public:
+    void DebugDump(const char * title) const
+    {
+#ifdef DEBUG
```

Comment:
What is DEBUG?  This is not something we have defined in our build system.

This bit should probably be done in a separate PR instead of it being sort of stuck in the middle of this big change.

Reply:
addressed in latest